src/app/parcels/add/AddParcelForm.tsx

```diff
@@ -88,6 +88,19 @@ const mergeDateAndTime = (date: Date, time: Date): Date => {
     );
 };
 
+const collectionCentreToKey = {
```

Comment:
Should this mapping be stored in the database? Feels a little weird to have it hard-coded here.

Reply:
Agreed

Reply:
this is chloe's ticket to do - this is just the quickest implementation I could do to get it working!